src/pages/RequestCallPage.js

```diff
@@ -65,22 +74,27 @@ class RequestCallPage extends Component {
     onSubmit() {
         this.setState({isLoading: true});
         if (!this.state.firstName.length || !this.state.lastName.length) {
-            Growl.show(this.props.translate('requestCallPage.growlMessageEmptyName'), CONST.GROWL.ERROR);
+            Growl.success(this.props.translate('requestCallPage.growlMessageEmptyName'));
```

Comment:
Doesn't look like `success` is an existing function for growl yet, same for other usage below

Reply:
Ah duh 😓 just assumed it existed since error was there too. I'll go ahead and implement it